Resources/App_GlobalResources/List_BannedPasswords-0.nl-NL.resx

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
This appears to be a "Portal" specific file, I don't think this should be as part of the language pack

Reply:
I did some more testing. The -0 .resx files are not in the distribution LP, but are created as soon as you have installed DNN and start the site. I will remove them from the PR